base/Dockerfile

```diff
@@ -14,28 +9,29 @@ VOLUME /root/.config/borg
 VOLUME /root/.ssh
 VOLUME /root/.cache/borg
 RUN apk add --update --no-cache \
-    tzdata \
-    sshfs \
-    openssl1.1-compat \
-    fuse \
+    bash \
```

Comment:
This is only because I sorted the packages alphabetically to make it easier to visually grep